lib/task-graph.js

```diff
@@ -165,6 +165,22 @@ function taskGraphFactory(
         });
     };
 
+   /*
+    *This function is help to get the task name for the task graph with
+    * without API path
+    */ 
+    var getTaskName = function(taskName){
+        // check if there is "/" in the task name ,if yes 
+        // it may have the api path before task name
+        var contains = taskName.indexOf("\/");
```

Comment:
temporary name could be isSlashContained. I think this is your purpose, it's more straightford than a verb 'contain